dts/arm/nuvoton/npcx/npcx7/npcx7-alts-map.dtsi

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
```

Comment:
nit: 2021

Reply:
Hi @yperess:
The npcx7 dts files such as npcx7-alts-map.dtsi, npcx7-espi-vws-map.dtsi and so on are moved from npcx to npcx/npcx7 folder. Do I need to treat them as new files?

Reply:
Ah, my mistake, the PR layout makes it unclear. It looked like they were new files added. Ignore.